storage/remote/codec.go

```diff
@@ -80,10 +81,30 @@ func EncodeReadResponse(resp *prompb.ReadResponse, w http.ResponseWriter) error
 	return err
 }
 
+func toWriteRequest(samples []tsdb.RefSample, series map[uint64][]prompb.Label) prompb.WriteRequest {
+	req := prompb.WriteRequest{
+		Timeseries: make([]prompb.TimeSeries, 0, len(samples)),
+	}
+
+	for _, s := range samples {
+		ts := prompb.TimeSeries{
+			Labels: series[s.Ref],
+			Samples: []prompb.Sample{
+				prompb.Sample{
+					Value:     float64(s.V),
+					Timestamp: int64(s.T),
+				},
+			},
+		}
+		req.Timeseries = append(req.Timeseries, ts)
+	}
+	return req
+}
+
 // ToWriteRequest converts an array of samples into a WriteRequest proto.
-func ToWriteRequest(samples []*model.Sample) *prompb.WriteRequest {
-	req := &prompb.WriteRequest{
-		Timeseries: make([]*prompb.TimeSeries, 0, len(samples)),
+func ToWriteRequest(samples []*model.Sample) prompb.WriteRequest {
```

Comment:
Is this still used, or can it be removed?  

If its still used, make it `[]model.Sample`.

Reply:
Yeah looks like its unused.